### ISSUE TITLE
Admin interface to send email to all subscribers

### ISSUE THREAD

**Issue by danielhers** [hasadna/anyway#251]
Depends on #233.
Once we have subscribers in the DB, we need to be able to email them about new features, preferably without having to run a SQL query manually.

**Comment by aviklai** [comment]
Through what email client? some smtp client?

**Comment by danielhers** [comment]
I don't know. Maybe Heroku supply a plug-in. Even a list of addresses would
be nice to start with.

**Comment by danielhers** [comment]
Flask admin sounds good, I think.

**Comment by danielhers** [comment]
About the smtp account - can we use gmail (anyway.feedback)?
@galraij can we use a `@hasadna.org.il` address or something?

**Comment by galraij** [comment]
I think anyway@anyway.co.il will the the best. And yes, it's gmail.



‫ב-17 במאי 2015, בשעה 09:27, ‏‏Daniel Hershcovich ‏<notifications@github.com> כתב/ה:‬

**Comment by samuelregev** [comment]
Gmail is not great when sending a a lot of emails, we might marked as spammers.
I recommend using SendGrid, you can specify what email will be shown as sender and there free account have limit of 400 emails a day which I believe is enough for start.
and they have add-on for heroku - https://devcenter.heroku.com/articles/sendgrid

**Comment by danielhers** [comment]
I added the plugin on Heroku.

**Comment by danielhers** [comment]
@aviklai What is still required to complete this? Login credentials for admin?

**Comment by aviklai** [comment]
Basicaly, It is completed. It just need to be tested with a real sendgrid account.
About login credentials for admin - it's already implemented.

**Comment by danielhers** [comment]
We have SendGrid enabled on our Heroku now.
You can login at http://oway.org.il/admin/login/ and try :)

**Comment by galraij** [comment]
Can someone send me (via email) the username and password? 

It look great from the outside :)



‫ב-25 במאי 2015, בשעה 16:15, ‏‏Daniel Hershcovich ‏<notifications@github.com> כתב/ה:‬

**Comment by aviklai** [comment]
I don't have it either.  I don't know if an admin user name and password were created at all. If not, I will do it today, after I figure out how to use heroku :)

**Comment by danielhers** [comment]
@njenia is this ready for use? Is there documentation on how to send the emails?

**Comment by aviklai** [comment]
The basic functionality of sending emails is working with sengrid. Currently it supports only sending plain text without file attachments.

**Comment by danielhers** [comment]
Great. If more enhancement are needed, open a separate issue.
